### ISSUE TITLE
Fate \Grand Order v.1.20.0 Force close

### ISSUE THREAD

**Issue by AkagiShiroe** [wDCat/ANRC#65]
hey could you fix this? i allready try with old 1.17.3 version its working but with this new version its just make it force close.

**Comment by wDCat** [comment]
What's version of ANRC are you using?

**Comment by AkagiShiroe** [comment]
latest beta in xposed 0.0.88(in ANRC about its readed as 0.89 Preview). allready try with/without "AG Patch". if i run old version first (1.17.3) then force it patched to newest (1.20.0) while ANRC mappig fake env running it's still working without giving any "force close".

**Comment by wDCat** [comment]
I may need more time to fix it._(:зゝ∠)_

Can you tell me the detail about how to patch the fate/go to 1.20.0~

**Comment by AkagiShiroe** [comment]
this is a silly way to be know with, heres the detail: (im using Nox and sorry for bad english)
1.first run Fate\ GO 1.17.3 version from ANRC with AG Patch(its should be working and prompth update)
2.then close FGO from recent task by swiping mouse on it
3. install newest APK then run it from ANRC without doing anything
(ENV. PROP Map status is still Mapping..)
the root are bypassed succesfully with your fake env. but im speculating the current method aren't compatible with F\GO Newest 1.20.0 since when Fake env. prop are stopped after that it will revert to "Force Close" problem due failure injection.

**Comment by RinFallen** [comment]
Hope you make it done soon :D I very appreciate it.

**Comment by Normiegetout** [comment]
Thank you for your hard work, is there any way to support you?

**Comment by dicedlemming** [comment]
I turned on logging, made it crash while launching f/go 1.20.0, and then exported it.

Thank you

[DLOG_1489776202836.txt](https://github.com/wDCat/ANRC/files/851473/DLOG_1489776202836.txt)

**Comment by manaphy0003** [comment]
*prays for fixed b4 event ends*

**Comment by wDCat** [comment]
try v0.0.90.1.
I have just update it to xposed repo.

**Comment by dicedlemming** [comment]
works on first run! thank you!

**Comment by RinFallen** [comment]
It's work :D Thanks you so much bro !

**Comment by Symmentry** [comment]
excuse me. i am getting this message after i updated ANRC can someone help me with this ? 
![image](https://cloud.githubusercontent.com/assets/20230210/24070938/aa1eb068-0c01-11e7-857d-95ecb23aadf0.png)

**Comment by Zeztia** [comment]
Error 92 is Modified apk / xposed, I got the same error but I solved it deleting the game  and installing again and ANCR I remove the F/GO app and put it again .

**Comment by Symmentry** [comment]
the hell? now im getting errorcodode==60. should i just re-download everything or something just to be safe?

**Comment by DaveHimura** [comment]
Hi all, my game was working fine, till it started crashing, so i downloaded the new version of ANRC, but now it gives me error 60. I've tried reinstalling all, old back up, but nothing working, would be grateful if someone can help me

**Comment by RinFallen** [comment]
[https://www.youtube.com/edit?o=U&video_id=ZxfNF4iUC-o](url)

**Comment by RinFallen** [comment]
[https://www.youtube.com/watch?v=ZxfNF4iUC-o](url)

**Comment by DaveHimura** [comment]
Sad, i cant make it work, silly error 60

**Comment by zhurai** [comment]
I'm getting error 60 too with the new ANRC version (v0.0.90.1)

**Comment by dicedlemming** [comment]
Try this.

![errors](https://cloud.githubusercontent.com/assets/13689396/24074363/86641e7a-0bdd-11e7-8d12-98f7ac3c58bf.jpg)

**Comment by zhurai** [comment]
error 60 fix listed above by @dicedlemming fixed the error here

thanks!

**Comment by DaveHimura** [comment]
Thanks all, got it working, bye bye error 60 :D

**Comment by shuyafaye** [comment]
Guys, when I start F/GO the screen went while for a while and then it just went auto-close without any error message appearing. I could run the game in non-xposed mode flawlessly before, this just happened yesterday and now I'm confused. The game version is 1.2.0 and I'm using the latest ANRC version.

**Comment by wDCat** [comment]
@shuyafaye try to backup account files and uninstall fate/go(including remove /mnt/sdcard/Android/data/com.aniplex.fategrandorder),then reinstall it.

**Comment by shuyafaye** [comment]
Thanks for the reply, @wDCat , I uninstalled and deleted the directory. I reinstalled the game through QooApp and used ANRC to start it again. The steps I took:

1. Installed F/GO version 1.20.0, when started, error 60 appears.
2. Open ANRC, add F/GO into App List
3. Open Settings, activate Non-Xposed Mode Switch. 
4. The app showed white screen when started for about 30 secs and forced closed without any error code. (white screen)
5. I went back to Settings and tried a few method again
 - Env. prop map > Regenerate the file
 - Support Lib > Reinstall libs
 - /system/bin/pm > Restore
 - SELinux > Disabled
6. The game won't start and still forced closed. 

Did I do anything wrong?

**Comment by maxiplier** [comment]
Yeah, same error going, i had both, fixed 60 and got 92, restored it and yet keep getting the same error, re-installed, error still progress, erased everything, and still going.. hope it can be fix soon.
